cmd/translate/main_grpc_integration_test.go

```diff
@@ -419,6 +419,20 @@ func Test_ListServices_gRPC(t *testing.T) {
 
```

Comment:
I am not quite sure about the scope of this PR, but shouldn't there be tests for http requests as well?

Reply:
+1 Add REST integration tests

Reply:
I wrote rest integration tests for updateMessages and for listMessages tests were already there but with the name GetMessages